bin/xo-tp-python

```diff
@@ -1,41 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
Similar comment to the one on `bin/xo` - If `xo-tp-python` isn't included in the `sawtooth` metapackage, the XO TP steps in several procedures are broken. For example, see Step 8 in "Using Ubuntu for Your Development Environment" (App Dev Guide): https://sawtooth.hyperledger.org/docs/core/nightly/master/app_developers_guide/ubuntu.html#step-8-start-the-transaction-processors